packages/components/src/Chips/InternalChipDismissible/InternalChipDismissibleInput.tsx

```diff
@@ -57,12 +56,6 @@ export function InternalChipDismissibleInput(props: ChipDismissibleInputProps) {
     update?.();
   }, [allOptions, menuOpen, update, options]);
 
-  useEffect(() => {
-    handleDebouncedSearch(searchValue, options);
-
-    return handleDebouncedSearch.cancel;
-  }, [searchValue, options]);
```

Comment:
while we should avoid doing this because it would be calling the search at times where we would not want to call the search, I'm not confident this isn't going to re-introduce the problem it was added to address